src/test/java/org/broadinstitute/hellbender/tools/IndexFeatureFileIntegrationTest.java

```diff
@@ -169,7 +169,7 @@ public void testVCFGZIndex_inferredName(){
         checkIndex(index, Arrays.asList("1", "2", "3", "4"));
     }
 
-    @Test(expectedExceptions = UserException.MalformedFile.class)
```

Comment:
@lbergelson and I have lamented how many of our tests expect non-subclassed UserExceptions and then get called with missing input files and everything passes.  I'd be happier if IndexFeatureFile threw a more specific UserException, but I understand you're lumping together a few different cases.  Maybe add a UserException.TribbleParseError or something similarly named?

Reply:
Unfortunately the underlying cause here can be anything from "file is malformed" to "codec does not support tabix", and it all comes out of htsjdk as a `TribbleException`. I thought about using `UserException.MalformedFile` here, which will be accurate 90%+ of the time, but there is that other 10%...

Reply:
Over time every test that catches raw UserException breaks and starts to pass incorrectly due to changes in the command line inputs.

Reply:
Yeah, it's hard when all we know is "Something is wrong"

Reply:
But I can switch to `UserException.MalformedFile` all the same, if you prefer.

Reply:
I would prefer UserException.MalformedFile to vanilla UserException.

Reply:
@lbergelson Again, the problem is in htsjdk in lumping all tribble errors together. Our choices in GATK are to throw a more-specific but sometimes-completely-inaccurate exception, or a more general exception with exposure of the underlying error message.

Reply:
Fixing htsjdk errors would be a nice benefit of redoing things in htsjdk, although it's going to be a big undertaking I think.

Reply:
How about I introduce a `UserException.FailureToIndexFile` exception here -- that would be accurate and address your concerns about tests erroneously passing.

Reply:
I'm actually more in favor of throwing the more general exception with the exposure of the underlying issue.  It's just nasty for testing.

Reply:
Good idea!

Reply:
I like it.